### PR TITLE
feature playlist

### PR DESCRIPTION
**Pull request by itcomesinwaves**

### CODE REVIEW

File: client/src/routes/playlists/playlist.component.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  
+    <h2>Search for Music (scary) </h2>
+    <input value={currentSearch}  onChange={handleSearchChange} placeholder='type for songs....'/>
+    <button onClick={handleSubmit} >BOO!</button>
+
+    { videos.map((video: Video) => {
+      return (
+        <div key={video.videoId} className="card" style={{width: '18rem', backgroundColor:'black'}}>
+        {/* <video src="video.mp4"></video> */}
```
Comment (iamdestinos) [inline client/src/routes/playlists/playlist.component.tsx:54]: Unsure what the point of this commented video element is? playlist component line 54

File: client/src/routes/playlists/playlist.component.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      )
+    }) }
+    </div>
+    <div className='col'>
+
+    <h2>Playlist...</h2>
+    { playlist.length > 0 ? playlist.map((video) => {
+      return (
+        <div key={video.videoId} className="card" style={{width: '18rem', backgroundColor:'black'}}>
+        {/* <video src="video.mp4"></video> */}
```
Comment (iamdestinos) [inline client/src/routes/playlists/playlist.component.tsx:72]: whats the point of this? line 72

Comment (itcomesinwaves) [reply client/src/routes/playlists/playlist.component.tsx:72]: that's gonna be the source of the actual video to play, but I don't have that particular type of data from the api yet

File: client/src/routes/playlists/playlist.component.tsx (hunk 1 above)
Comment (itcomesinwaves) [reply client/src/routes/playlists/playlist.component.tsx:54]: Same as 72

**Review by iamdestinos — APPROVED** [review]
Should probably put something in the empty files like a comment on if its supposed to be empty due to being not yet implemented. Approved otherwise

**Review by jimenezfede — APPROVED** [review]

**Pull request merged**